sync_pr_to_gitlab/github_pr_to_internal_pr.py

```diff
@@ -171,13 +170,14 @@ def main():
         print('Not running in GitHub action context, nothing to do')
         return
 
-    if not os.environ['GITHUB_REPOSITORY'].startswith('espressif/'):
-        print('Not an Espressif repo!')
-        return
+    # TODO: Commented out for testing
```

Comment:
Please don't forget uncomment

Reply:
We are currently testing it with @laukik-hase. There haven't been significant changes from the original code, so it should be safe. However, it's better to test it now than to deal with potential issues under time pressure later.

This will be uncommented before the final merge. Thanks!